x/twap/api_test.go

```diff
@@ -113,100 +158,201 @@ func (s *TestSuite) TestGetArithmeticTwap() {
 	tests := map[string]struct {
 		recordsToSet []types.TwapRecord
 		ctxTime      time.Time
-		input        getTwapInput
-		expTwap      sdk.Dec
+		input        []getTwapInput
+		expTwap      []sdk.Dec
 		expectError  error
 	}{
 		"(1 record) start and end point to same record": {
 			recordsToSet: []types.TwapRecord{baseRecord},
 			ctxTime:      tPlusOneMin,
 			input:        makeSimpleTwapInput(baseTime, tPlusOne, quoteAssetA),
-			expTwap:      sdk.NewDec(10),
+			expTwap:      []sdk.Dec{sdk.NewDec(10)},
+		},
+		"(1 record, three asset pool) start and end point to same record": {
```

Comment:
Re: discussion offline. I think I see what you are saying - this is a new multi-asset test added.

I think I misinterpreted that this is a modified test case because it is still named as "1 record. I suggest renaming this to "3 records at time t".

"1 record" does not really apply because we have 3. They are all just at the same time for multiple denom pairs.

Reply:
Yeah I kept debated changing it to 3 record.. since this was unclear though I will for sure change this now, thanks!